src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -283,6 +298,8 @@ namespace OpenMS
     /// Assignment operator (not implemented)
     SpectrumMetaDataLookup& operator=(const SpectrumMetaDataLookup&);
 
+    /// adds Ion matches to identifcations
+    static void addIonMatches_(PeptideIdentification& pi, const MSSpectrum<Peak1D> &spec);
```

Comment:
this is not lookup but reannotation with our algorithms. @hendrikweisser would you place this in this class?